rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc

```diff
@@ -10,6 +12,26 @@
 | 30 GB of disk space | 30 GB (SSD) of disk space
 |===
 
+* To run more than one process in parallel, when using Secure Sessions, you need to add 2 GB of RAM plus the RAM needed to run the process for each parallel run to these recommended values.
++
+For example, to run two different processes in parallel:
++
+Process 1 requires one GB of RAM and runs 10 times in parallel.
```

Comment:
Suggest making the Process sentences subbullets.
```
+
--
** Process 1 ...
** Process 2 ...
--
```

Reply:
Thanks for this tip with the -- I tried in vain to format it this way :)